Custom Remedy AR (#31436)
demisto/content: Packs/Remedy_AR/ReleaseNotes/1_0_9.md

#### Integrations

##### BMC Remedy AR

> - Fixed an issue where the **Test Button** failed.

Reviewer: ```suggestion
- Fixed an issue with the authentication of the integration.
```
